lib/src/components/NxCombobox/NxCombobox.tsx

```diff
@@ -51,14 +51,14 @@ function NxComboboxRender<T extends string | number | DataItem<string | number,
       } = props,
       previousValue = usePrevious(value),
       newAttrs = omit(['trimmedValue'], attrs),
-
+      [hiddenBySelection, setHiddenBySelection] = useState(false),
```

Comment:
I think a comment should be added explaining what this state variable is for

Reply:
> I think a comment should be added explaining what this state variable is for

Good idea, added here: [e616eca](https://github.com/sonatype/sonatype-react-shared-components/pull/876/commits/e616eca270fac3c4c24f0649b37d178c0f806f04)